21.3-1/integration_tests/azure_hirsute_integration_tests_rerun.txt

```diff
@@ -0,0 +1,1081 @@
+2021-09-10 13:56:57 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
+2021-09-10 13:56:58 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
+2021-09-10 13:56:59 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
+2021-09-10 13:57:00 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
+2021-09-10 13:57:01 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
+2021-09-10 13:57:02 INFO      paramiko.transport:transport.py:1819 Connected (version 2.0, client OpenSSH_8.4p1)
+2021-09-10 13:57:02 INFO      paramiko.transport:transport.py:1819 Authentication (publickey) successful!
+2021-09-10 13:57:05 INFO      pycloudlib.instance:instance.py:175 executing: cloud-init status --wait --long
+Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7f1d8b713780>
+2021-09-10 13:57:18 INFO      integration_testing:clouds.py:173 Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7f1d8b713780>
+2021-09-10 13:57:18 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init --version'
+cloud-init version: /usr/bin/cloud-init 21.2-3-g899bfaa9-0ubuntu2~21.04.1
+2021-09-10 13:57:19 INFO      integration_testing:clouds.py:179 cloud-init version: /usr/bin/cloud-init 21.2-3-g899bfaa9-0ubuntu2~21.04.1
+2021-09-10 13:57:19 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'grep serial /etc/cloud/build.info'
+image serial: 20210908
+2021-09-10 13:57:19 INFO      integration_testing:clouds.py:183 image serial: 20210908
+2021-09-10 13:57:19 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c hostname
+2021-09-10 13:57:19 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c cloud-id
+2021-09-10 13:57:20 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cat /run/cloud-init/result.json'
+2021-09-10 13:57:20 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cat /etc/netplan/50-cloud-init.yaml'
+2021-09-10 13:57:20 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c systemd-analyze
+2021-09-10 13:57:20 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'systemd-analyze blame'
+2021-09-10 13:57:21 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init analyze show'
+2021-09-10 13:57:22 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init analyze blame'
+2021-09-10 13:57:22 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cat /var/log/cloud-init.log'
+There were errors/warnings/tracebacks pre-upgrade. Any failures may be due to pre-upgrade problem
+2021-09-10 13:57:22 WARNING   integration_testing.test_upgrade:test_upgrade.py:84 There were errors/warnings/tracebacks pre-upgrade. Any failures may be due to pre-upgrade problem
```

Comment:
Do we know on hirsute what these pre-existing tracebacks were?

Reply:
```
2021-09-14 16:42:40,580 - cc_disk_setup.py[DEBUG]: setting up filesystems: [{'filesystem': 'ext4', 'device': 'ephemeral0.1'}]
2021-09-14 16:42:40,580 - cc_disk_setup.py[DEBUG]: ephemeral0.1 is mapped to disk=/dev/disk/cloud/azure_resource part=1
2021-09-14 16:42:40,580 - cc_disk_setup.py[DEBUG]: Creating new filesystem.
2021-09-14 16:42:40,581 - subp.py[DEBUG]: Running command ['udevadm', 'settle'] with allowed return codes [0] (shell=False, capture=True)
2021-09-14 16:44:40,593 - util.py[DEBUG]: Creating fs for /dev/disk/cloud/azure_resource took 120.012 seconds
2021-09-14 16:44:40,593 - util.py[WARNING]: Failed during filesystem operation
Unexpected error while running command.
Command: ['udevadm', 'settle']
Exit code: 1
Reason: -
Stdout: 
Stderr: 
2021-09-14 16:44:40,593 - util.py[DEBUG]: Failed during filesystem operation
Unexpected error while running command.
Command: ['udevadm', 'settle']
Exit code: 1
Reason: -
Stdout: 
Stderr: 
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cloudinit/config/cc_disk_setup.py", line 157, in handle
    util.log_time(logfunc=LOG.debug,
  File "/usr/lib/python3/dist-packages/cloudinit/util.py", line 2348, in log_time
    ret = func(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/cloudinit/config/cc_disk_setup.py", line 886, in mkfs
    assert_and_settle_device(device)
  File "/usr/lib/python3/dist-packages/cloudinit/config/cc_disk_setup.py", line 765, in assert_and_settle_device
    util.udevadm_settle()
  File "/usr/lib/python3/dist-packages/cloudinit/util.py", line 2608, in udevadm_settle
    return subp.subp(settle_cmd)
  File "/usr/lib/python3/dist-packages/cloudinit/subp.py", line 293, in subp
    raise ProcessExecutionError(stdout=out, stderr=err,
cloudinit.subp.ProcessExecutionError: Unexpected error while running command.
Command: ['udevadm', 'settle']
Exit code: 1
Reason: -
Stdout: 
Stderr: 
2021-09-14 16:44:40,636 - handlers.py[DEBUG]: finish: init-network/config-disk_setup: SUCCESS: config-disk_setup ran successfully
```

You can see it if you launch a hirsute latest image on Azure

Reply:
It's probably worth us filing a bug on this issue for tracking? We can tackle that separately though. This is unrelated to the current SRU.

Reply:
Thanks. I filed https://bugs.launchpad.net/cloud-init/+bug/1943715